app/models/course_user.rb

```diff
@@ -63,6 +75,17 @@ class CourseUser < ActiveRecord::Base
   scope :students, -> { where(role: roles[:student]) }
 
   scope :with_course_statistics, -> { all.calculated(:experience_points, :achievement_count) }
+  scope :ordered_by_experience_points, (lambda do
+    all.calculated(:experience_points).order('experience_points DESC')
```

Comment:
As mentioned above, have to use raw sql because experience_points is a calculated attribute.